test/resource/matter-test-nostatic.zap

```diff
@@ -0,0 +1,2397 @@
+{
+  "fileFormat": 2,
+  "featureLevel": 108,
+  "creator": "zap",
+  "keyValuePairs": [
+    {
+      "key": "commandDiscovery",
+      "value": "1"
+    },
+    {
+      "key": "defaultResponsePolicy",
+      "value": "always"
+    },
+    {
+      "key": "manufacturerCodes",
+      "value": "0x1002"
+    },
+    {
+      "key": "generateStaticTemplates",
```

Comment:
Does this take precedence over the templates.json -> "generateStaticTemplates": "true" value?